engine/dumps/bank03.asm

```diff
@@ -2353,7 +2353,7 @@ Functione2b0:
 	scf
 	ret
 .sub_e2c4
-	ld hl, Texte2dc
+	ld hl, Texte2dc ; failed deposit last mon
```

Comment:
Don't describe the contents of texts either. The label should be descriptive, and any further deliberation/translation should be done where the text *actually is*, rather than every time it's used.